doc/developer-guides/hld/virtio-console.rst

```diff
@@ -198,3 +201,35 @@ The File backend only supports console output to a file (no input).
 #. Add the console parameter to the guest OS kernel command line::
 
       console=hvc0
+
+SOCKET
+======
+
+The virtio-console socket-type can be set as socket server or client. Device model will
+create an unix domain socket if appointed the socket_type as server, then server VM or
+another user VM can bind and listen for communication requirement. If appointed to
+client, please make sure the socket server is ready prior to launch device model.
+
+1. Add a pci slot to the device model (``acrn-dm``) command line, adjusting
+   the ``</path/to/file.sock>`` to your use case in the VM1 configuration::
+
+      -s n,virtio-console,socket:socket_file_name=</path/to/file.sock>:server
+
+#. Add a pci slot to the device model (``acrn-dm``) command line, adjusting
+   the ``</path/to/file.sock>`` to your use case in the VM2 configuration::
+
+      -s n,virtio-console,socket:socket_file_name=</path/to/file.sock>:client
+
+#. Login to VM1, connect to the virtual port(vport1p0, 1 is decided
+   by front-end driver)::
+
+      minicom -D /dev/vport1p0
+
```

Comment:
1, Please change to #minicom -D /dev/vport1p0.
2, Please change the style to console style, please refer https://projectacrn.github.io/latest/developer-guides/hld/random-virt-hld.html